alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -64,7 +64,8 @@ func resourceAlkiraConnectorAzureVnet() *schema.Resource {
 				Optional: true,
 				Default:  "VNET_GATEWAY",
 				ValidateFunc: validation.StringInSlice([]string{
-					"VNET_GATEWAY", "VNET_PEERING"}, false),
+					"VNET_GATEWAY", "VNET_PEERING",
+				}, false),
```

Comment:
what's changed here except new line?

Reply:
sorry, this is caused by "gofumpt" formatting.

Reply:
lol, we used the standard `gofmt`, so it should be okey. It's funny that the tool says that it won't generate diff from `gofmt`. Hehe.